cli/envision.py

```diff
@@ -22,12 +22,12 @@
 from envision.server import run
 
 
-@click.group(name="envision")
+@click.group(name="envision", help="Start an Envision server. See `scl envision start --help` for further options")
```

Comment:
We do not need to redirect to a specific command here.

```suggestion
@click.group(name="envision", help="Commands to utilize an Envision server. The Envision web server is used for visualization purposes. See `scl envision COMMAND --help` for further options.")
```